src/main/java/com/users/mappers/UserMapper.java

```diff
@@ -0,0 +1,16 @@
+package com.users.mappers;
+
+import com.users.dto.UserDto;
+import com.users.entities.User;
+import org.mapstruct.Mapper;
+import org.mapstruct.Mapping;
+import org.mapstruct.MappingTarget;
+
+
+@Mapper(componentModel = "spring")
+public interface UserMapper {
+    UserDto toDto(User user);
```

Comment:
lasa un spatiu intre metode sa fie mai aerisit

Reply:
adica intre adnotare si interfata trebuie sa las un spatiu?

Reply:
nu, intre declararea interfetei public interface UserMapper{ si metoda toDto sa fie un spatiu, si la fel intre to Dto si toEntity, nu e un must, dar arata mai aerisit

Reply:
gata, asa am facut